modzy/models.py

```diff
@@ -382,8 +382,58 @@ def get_models(self, model_id=None, author=None, created_by_email=None, name=Non
         json_list = self._api_client.http.get('{}?{}'.format(self._base_route, urlencode(body)))
         return list(Model(json_obj, self._api_client) for json_obj in json_list)
 
+    def edit_model_metadata(self, model_id, model_version, long_description=None, technical_details=None, 
+                            performance_summary=None, performance_metrics=None, input_details=None, output_details=None):
+
+        '''
+        Edit a model's metadata after it is deployed
+
+        Args:
+            model_id (str): Model identifier of model to edit
+            model_version (str): Model version of model to edit
+            long_description (str): Description to appear on model biography page
+            technical_details (str): Technical details to appear on model biography page. Markdown is accepted
+            performance_summary (str): Description providing model performance to appear on model biography page
+            performance_metrics (List): List of arrays describing model performance statistics
+            input_details (List): List of dictionaries describing details of model inputs
+            output_details (List): List of dictionaries describing details of model outputs
+
+        Returns:
+            dict: Metadata of newly edited model information including formatted URL to newly deployed model page.
+        Raises:
+            ApiError: A subclass of ApiError will be raised if the API returns an error status,
+                or the client is unable to connect.        
+        '''
+
+        # validate model version exists    
+        try:
+            json_obj = self._api_client.http.get('{}/{}/versions/{}'.format(self._base_route, model_id, model_version))
+            _ = ModelVersion(json_obj, self._api_client)
```

Comment:
What's the purpose of this line? Just to see if the object returned can decode into a ModelVersion object without throwing an exception?